src/locales/de_DE.json

```diff
@@ -231,197 +253,231 @@
 "label.action.stop.systemvm.processing": "System-VM wird gestoppt....",
 "label.action.take.snapshot": "Schnappschuss erstellen",
 "label.action.take.snapshot.processing": "Schnappschuss wird gemacht....",
-"label.action.unmanage.cluster": "Vernachl\u00e4ssige Cluster",
+"label.action.template.permission": "Update Template Erlaubnis",
+"label.action.template.share": "Update Template Verteilung",
+"label.action.unmanage.cluster": "Vernachlässige Cluster",
```

Comment:
Clusterverwaltung aufgeben

Reply:
@onitake both are not nice. do we have a better word for "aufgeben"?